### ISSUE TITLE
Merge from Develop to master has broken multinode provisioning on GCE

### ISSUE THREAD

**Issue by hiremaga** [EMCECS/ECS-CommunityEdition#109]
We noticed that we can no longer provision an ECS cluster since this commit was merged: cb0935dc6b2b753448cf2906b93f412c09555dab

This is the associated error:

```
[root@ecs1 ecs-multi-node]# python step1_ecs_multinode_install.py --ips $node1 $node2 $node3 $node4 --hostnames ecs1 ecs2 ecs3 ecs4 --disks sdb --eth
adapter eth0
  File "step1_ecs_multinode_install.py", line 544
    os.system("docker "+' '.join(DockerCommandLineFlags)+" exec -t  ecsmultinode tar zxvf master -C /tmp")        os.system("docker "+' '.join(Docker
CommandLineFlags)+" exec -t -i ecsstandalone bash -c \"cd /tmp/kennethreitz-requests-* && python setup.py install\"")
```

Reverting to 72305b4b929c3c652c6162d9973296e4bd163f68 appears to work.

~ Joel (@freedmanpivotal) & Abhi from Pivotal

**Comment by jasoncwik** [comment]
@akramh do you have time to update support for GCE?

**Comment by APeter0807** [comment]
Resolved as of f91537f
